rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -252,7 +252,7 @@ public MavenMetadata downloadMetadata(GroupArtifactVersion gav, @Nullable Resolv
                     try {
                         MavenMetadata derivedMeta = deriveMetadata(gav, repo);
                         if (derivedMeta != null) {
-                            Counter.builder("rewrite.maven.derived.metatdata")
+                            Counter.builder("rewrite.maven.derived.metadata")
```

Comment:
scary? (https://grep.app doesn't see any [consumers](https://grep.app/search?q=rewrite.maven.derived.metatdata)..., otoh, it isn't a [rare typo](https://grep.app/search?q=metatdata))

Reply:
@kmccarp Do these metrics end up anywhere that would break with this spelling fix?

Reply:
> @kmccarp Do these metrics end up anywhere that would break with this spelling fix?

I don't believe there is a dashboard with this on it in grafana, so all good 👍